# Generated by Django 2.2.7 on 2019-12-17 06:38

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Error_task',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('c_type', models.CharField(choices=[('equal_zero', '为0'), ('equal_null', '为空'), ('equal_neg', '为负'), ('delay', '数据延迟')], max_length=32)),
                ('s_id', models.CharField(max_length=32)),
                ('t_name', models.CharField(max_length=128)),
                ('c_name', models.CharField(max_length=128)),
                ('r_time', models.TimeField()),
                ('o_user_email', models.EmailField(max_length=128)),
                ('p_user_email', models.EmailField(max_length=128)),
                ('c_time', models.DateTimeField(auto_now_add=True)),
                ('u_time', models.DateTimeField(auto_now=True)),
            ],
            options={
                'verbose_name': '异常监控',
                'ordering': ['-c_time'],
                'verbose_name_plural': '异常监控',
            },
        ),
    ]
